shared/build/docker/container.go

```diff
@@ -144,3 +144,11 @@ func (c *ContainerService) RunDaemonPorts(image string, ports map[Port]struct{})
 	//map[3306/tcp:{}] map[3306/tcp:[{127.0.0.1 }]]
 	return c.RunDaemon(&config, &host)
 }
+
+func (c *ContainerService) RunDaemonHost(image string) (*Run, error) {
+	// setup configuration
+	config := Config{Image: image}
+	host := HostConfig{}
+	host.NetworkMode = "host"
```

Comment:
When using `--net=host` all ports are made available to the host system and an application can see/use then hosts network adapters.

Therefore we only launch the services this way when:

- This is not a PR
- The user has this in their YAML:

```
docker:
  net: host
```